docs/development/extensions-core/druid-lookups.md

```diff
@@ -31,12 +31,12 @@ The main goal of this cache is to speed up the access to a high latency lookup s
 Thus user can define various caching strategies or and implementation per lookup, even if the source is the same.
 This module can be used side to side with other lookup module like the global cached lookup module.
 
-To use this extension please make sure to  [include](../../development/extensions.md#loading-extensions) `druid-lookups-cached-single` as an extension.
+To use this extension please make sure to  [include](../extensions.md#loading-extensions) `druid-lookups-cached-single` as an extension.
```

Comment:
```suggestion
To use this extension [include](../extensions.md#loading-extensions) `druid-lookups-cached-single` as an extension.
```

Reply:
should this be something like " ... include `druid-lookups-cached-single` in the extension load list." so it isn't "this extension ...  as an extension."?